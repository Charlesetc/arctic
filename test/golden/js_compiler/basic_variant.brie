define main [
  define x (Hi 2)
]
