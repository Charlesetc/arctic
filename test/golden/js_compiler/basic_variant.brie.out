const global_functions = {1: {}};
function main() {
	return var x = {Hi: [2]};
}
console.log(main());
var test/golden/js_compiler/basic_variant.brie_main = new_closure(1)